tst/EnergyPlus/unit/IdfParser.unit.cc

```diff
@@ -0,0 +1,282 @@
+// IdfParser Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include "Fixtures/IdfParserFixture.hh"
+
+namespace EnergyPlus {
+
+	TEST_F( IdfParserFixture, decode ) {
```

Comment:
Yeah but now that I know the type defs are floating around I am always second guessing.  How do I know what you did in IdfParserFixture.hh?  You could have but a type def in there.  Yes once you learn the convention it might be fine, but people aren't going to implement the convention consistently.